src/Views/gagepage.html

```diff
@@ -59,6 +59,10 @@ <h4 class="modal-title">StreamStats Data-Collection Station Report</h4>
                                 <th scope="row">Agency</th>
                                 <td>{{vm.gage.agency.name}}</td>
                             </tr>
+                            <tr>
+                               <th scope="row">NWIS Period of Record</th>
```

Comment:
Change to "NWIS Discharge Period of Record" to improve clarity. Some gages do not have a valid StartDate or EndDate, even if one is shown on the NWIS page ([example](https://nwis.waterdata.usgs.gov/nwis/inventory/?site_no=02087140)) because that period of record is associated with a code other than 00060 (Discharge).

Reply:
Resolved in https://github.com/USGS-WiM/StreamStats/pull/1237/commits/5c3a49d9c5218fa91e08b702a0f06a0a7b904816